docs/rules/eventmap-params.md

```diff
@@ -70,6 +76,54 @@ Template.foo.events({
 
 ```
 
+#### Destructuring
+
+You can optionally allow destructuring the parameters.  
```

Comment:
Shouldn't this be "You can optionally **forbid** destructuring the parameters."? It's allowed by default isn't it?